### ISSUE TITLE
Some Manual Section(s) have Contents not in Section Table of Contents

### ISSUE THREAD

**Issue by timfenney** [NixOS/nixpkgs#213690]
## Problem

As I understand it, the Table of Contents _should_ show the locations of all contents in a Document (Section). However the NixOS Manual Section(s) (chapters) may have a Section TOC, whose first entry may link to a subsection well after the Section content has started.

As a User I find this strange, because I am accustomed to being able to click the first item in a TOC, and thereby skip to the actual content of the Document. If I do so when reading the NixOS Manual, I will skip more than the TOC, but also content which I would like to read.

![nixos-manual-skipped-content](https://user-images.githubusercontent.com/654400/215638347-ae4c942b-c060-4cc2-a870-f18db40098da.png)

## Checklist

- [x] checked [latest Nixpkgs manual] \([source][nixpkgs-source]) and [latest NixOS manual] \([source][nixos-source])
- [x] checked [open documentation issues] for possible duplicates
- [x] checked [open documentation pull requests] for possible solutions

[latest Nixpkgs manual]: https://nixos.org/manual/nixpkgs/unstable/
[latest NixOS manual]: https://nixos.org/manual/nixos/unstable/
[nixpkgs-source]: https://github.com/NixOS/nixpkgs/tree/master/doc
[nixos-source]: https://github.com/NixOS/nixpkgs/tree/master/nixos/doc/manual
[open documentation issues]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+label%3A%229.needs%3A+documentation%22
[open documentation pull requests]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+is%3Apr+label%3A%228.has%3A+documentation%22%2C%226.topic%3A+documentation%22

## Proposal

One fix for this would be to make sure that Sections with Subsections themselves linked from the aforementioned Section's TOC, contain no contents as children of the Section, i.e. that all contents of such a Section are contained in linked Subsections.

A second idea is to remove the inline TOC for a chapter. At least when the manual is viewed on a single HTML page, an inline chapter TOC doesn't seem to add much to me.

**Comment by timfenney** [comment]
@fricklerhandwerk what do you think?

**Comment by timfenney** [comment]
@fricklerhandwerk what do you think?

**Comment by fricklerhandwerk** [comment]
I agree that usability suffers here, but I think it's a minor problem. If we rendered the manual on separate pages as done for Nix with mdBook, the navigation would work differently and you'd actually need content for the top level section, because otherwise that page would be empty (as is currently still the case for the Nix manual).

It may be simpler to move the chapter TOC after the top section. Then it's clear that the top section serves as an introduction, and one would navigate to it using the chapter heading.
